core/clients/key_flow_test.go

```diff
@@ -34,7 +36,10 @@ const saKeyStrPattern = `{
 	"validUntil": "2024-03-22T18:05:41Z"
 }`
 
-var saKey = fmt.Sprintf(saKeyStrPattern, uuid.New().String(), uuid.New().String(), uuid.New().String())
+var (
+	saKey          = fmt.Sprintf(saKeyStrPattern, uuid.New().String(), uuid.New().String(), uuid.New().String())
+	testSigningKey = []byte("Test")
+)
```

Comment:
```suggestion
const (
	saKey          = fmt.Sprintf(saKeyStrPattern, uuid.New().String(), uuid.New().String(), uuid.New().String())
	testSigningKey = []byte("Test")
)
```

Reply:
They are not constant. The first one results from string formatting and the second because it is an array (it can be appended to, for example)

Reply:
Ah I though we could define them as `const`s and not allow changes